api/tests/unit/domain/usecases/find-competence-evaluations-by-assessment_test.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+const { expect, sinon, catchErr } = require('../../../test-helper');
+const findCompetenceEvaluationsByAssessment = require('../../../../lib/domain/usecases/find-competence-evaluations-by-assessment');
+const { UserNotAuthorizedToAccessEntity } = require('../../../../lib/domain/errors');
+
+describe('Unit | UseCase | find-competence-evaluations-by-assessment', () => {
+
+  const userId = 1;
+  const assessmentId = 2;
+  const assessmentRepository = { ownedByUser: _.noop() };
+  const competenceEvaluationRepository = { findByAssessmentId: _.noop() };
+
+  beforeEach(() => {
+    assessmentRepository.ownedByUser = sinon.stub();
+    competenceEvaluationRepository.findByAssessmentId = sinon.stub();
+  });
+
+  it('should throw an UserNotAuthorizedToAccessEntity error when user is not the owner of the assessment', async () => {
+    // given
+    assessmentRepository.ownedByUser.withArgs({ id: assessmentId, userId }).resolves(false);
+
+    // when
+    const error = await catchErr(findCompetenceEvaluationsByAssessment)({
+      userId,
+      assessmentId,
+      competenceEvaluationRepository,
+      assessmentRepository,
+    });
+
+    // then
+    expect(error).to.be.instanceOf(UserNotAuthorizedToAccessEntity);
+  });
+
+  it('should return the assessment competence-evaluations', async () => {
```

Comment:
```suggestion
  it('should return the assessment competence-evaluations when user is the owner of the assessment', async () => {
```
Je trouve cela plus claire par rapport it precedent

Reply:
Ok merci c'est fait